boards/nxp/frdm_mcxn947/board.c

```diff
@@ -6,6 +6,9 @@
 #include <zephyr/device.h>
 #include <fsl_clock.h>
 #include <fsl_spc.h>
+#if defined(CONFIG_SOC_MCXN947) && defined(CONFIG_ADC_MCUX_LPADC)
```

Comment:
Only the `CONFIG_ADC_MCUX_LPADC` needs to be present here- this file is specific to the MCXN947 SOC

Reply:
> Only the `CONFIG_ADC_MCUX_LPADC` needs to be present here- this file is specific to the MCXN947 SOC

Yes, this is mcxn947 specific file. I will update it.